### PR TITLE
Ns/415 identify novice riders

### PR DESCRIPTION
**Pull request by neal-bpm**
## Describe your changes

https://github.com/bikebrigade/dispatch/issues/415

Feature added to easily identify novice riders (those with fewer than 5 deliveries) on the campaign page using Star icon as an indicator.
star_icon_for_novice_riders

## Checklist before requesting a review

- [x] I have performed a self-review of my code - Yes
- [ ] If it is a core feature, I have added tests. - No
- [ ] Are there other PRs or Issues that I should link to here? - No
- [ ] Will this be part of a product update? If yes, please write one phrase
      about this update in the description above.

### CODE REVIEW

File: lib/bike_brigade_web/live/campaign_live/riders_list_component.html.heex [hunk 1]
```diff
@@ -67,6 +67,20 @@
...
             <% end %>
+            <%= if novice_participant?(rider) do %>
+              <.with_tooltip>
+                <Heroicons.star
+                  mini
+                  class="flex-shrink-0 w-4 h-4 mx-1 text-gray-500 justify-self-end"
+                />
+                <:tooltip>
+                  <div class="w-20">
+                    Novice rider
```
Comment (cdp-blackout) [inline lib/bike_brigade_web/live/campaign_live/riders_list_component.html.heex:78]:
```suggestion
                    New rider (less than 5 campaigns)
```

**Review by cdp-blackout — APPROVED** [review]
Looks good! I suggested a small copy improvement, otherwise good to ship!

File: lib/bike_brigade_web/live/campaign_live/riders_list_component.html.heex (hunk 1 above)
Comment (mveytsman) [inline lib/bike_brigade_web/live/campaign_live/riders_list_component.html.heex:78]: New rider (less than 5 campaigns)

**Review by mveytsman — APPROVED** [review]
Looks good! Suggested a small copy change!

**Pull request merged**